index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
```

Comment:
I don't think mapping creates compatibility, but it does enable consistent processing in graph databases.... concrete example, a security researcher might map stolen `vc+ld+json`, `vc+ld+jwt`, `id_token`, `access_token` and `mDoc` expressions to the data model (vc+ld+json) before performing a graph analysis of the stolen credentials.

It's not correct to say they are "compatible", but that can be "normalized" by the mapping, to facilitate, surveillance or risk management / threat analysis.

Reply:
Changing this language to "can be normalized" would make the statement a truism causing it to provide no value to the specification.  Anything that is transformed into vc+ld+json (a conforming document) can, by definition, be normalized into vc+ld+json whether the transformation follows the steps contained in the rest of this section or not.

Reply:
yeah, I don't think the language is good in either case... 

> transformation follows the steps contained in the rest of this section or not.

I don't think we are defining "steps to transform arbitrary content types to JSON-LD"... in this working group... are we?